File: Easy/Blackjack solver/Blackjack solver.cs
using System;
using System.Linq;
using System.Collections.Generic;

class Solution
{
    static void Main(string[] args)
    {
        List<string> bankCards = Console.ReadLine().Split(' ').ToList();
        List<string> playerCards = Console.ReadLine().Split(' ').ToList();

        int bankresult = calculatescore(bankCards);
        int playerresult = calculatescore(playerCards);

        if (playerCards.Count() == 2 && playerresult ==21 && bankresult ==21 && bankCards.Count() ==2)
            Console.WriteLine("Draw");
        else if (playerCards.Count() == 2 && playerresult ==21)
            Console.WriteLine("Blackjack!");
        else if (playerresult>21)
            Console.WriteLine("Bank");
        else if (bankresult>21)
            Console.WriteLine("Player");
        else if (bankresult<playerresult)
            Console.WriteLine("Player");
        else if (bankresult>playerresult)
            Console.WriteLine("Bank");
        else
            Console.WriteLine("Draw");
    }

    public static int calculatescore(List<string> a)
    {
        int[] scores = new int[a.Count];
        for (int i = 0; i < a.Count(); i++)
        {
            if (!int.TryParse(a[i], out int r))
            {
                if (a[i] == "A")
                    scores[i] = 11;
                else
                    scores[i] = 10;
            }
            else
                scores[i] = r;
        }
        while (scores.Sum() > 21 && Array.IndexOf(scores,11)!=-1)
            scores[Array.IndexOf(scores,11)] = 1;

        return scores.Sum();
    }
}

File: Easy/Blackjack solver/Blackjack solver.py
import sys
import math

def calculatescore(a):
    scores = []
    for i in a:
        if (i.isdigit()==False):
            if (i == "A"):
                scores.append(11)
            else:
                scores.append(10)
        else:
            scores.append(int(i))

    while sum(scores) > 21 and scores.count(11)>0:
        scores[scores.index(11)] = 1

    return sum(scores)

bank_cards = input().split(' ')
player_cards = input().split(' ')

bankresult = calculatescore(bank_cards)
playerresult = calculatescore(player_cards)

if len(player_cards) == 2 and playerresult == 21 and bankresult ==21 and len(bank_cards) == 2:
    print("Draw")
elif len(player_cards) == 2 and playerresult ==21:
    print("Blackjack!")
elif playerresult>21:
    print("Bank")
elif (bankresult>21):
    print("Player")
elif bankresult<playerresult:
    print("Player")
elif bankresult>playerresult:
    print("Bank")
else:
    print("Draw")